docs/deploying-cf-aws.MD

```diff
@@ -0,0 +1,374 @@
+
+# Create the bbl-user AWS account
+Having said that, the first thing that you will need to do is create an AWS account using the steps described [here](https://cloudfoundry.github.io/bosh-bootloader/getting-started-aws/).
+
+```bash
+pbpaste="{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "logs:*",
+                "elasticloadbalancing:*",
+                "cloudformation:*",
+                "iam:*",
+                "kms:*",
+                "route53:*",
+                "ec2:*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}"
+aws iam create-user --user-name "bbl-user"
+aws iam put-user-policy --user-name "bbl-user" \
+    --policy-name "bbl-policy" \
+    --policy-document "$(pbpaste)"
+aws iam create-access-key --user-name "bbl-user"
+```
+
+Store the access key and secret in a safe location, you will need to reference these in the next set of commands and in the future when you need to interact with the AWS resources.
+
+## Setup the bosh environment
+Before dweling into the bosh environment, make sure you have downloaded the latest `bbl`, `bosh` and `credhub` cli binaries. The `bbl` binary can be downloaded [here](https://github.com/cloudfoundry/bosh-bootloader/releases), while the `bosh` binary [here](https://github.com/cloudfoundry/bosh-cli/releases). Finally, the `credhub` binary can be downloaded [here](https://github.com/cloudfoundry/credhub-cli/releases).
+
+Once you're downloaded both binaries and they are in your path and ready to be used, run the following commands to export the environment variables to be reused in the installation. Replace the variables in `<>` with your specific values.
+
+```bash
+export access_key_id=<AWS_BBL_USER_ACCESS_KEY_ID>
+export access_key_secret=<AWS_BBL_USER_ACCESS_KEY_SECRET>
+export region=<AWS_REGION>
+export ENV_NAME=aws
+export SYSTEM_DOMAIN=bosh-$ENV_NAME.com
+```
+
+## Generate a self signed certificates for for Load Balancer:
+The AWS deployment deploys a Load Balancer as part of the ingress communication between the client and the Cloud Foundry infrastructure. Here we will create a self signed certificate that matches the domain we plan to use:
+
+```bash
+openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem \
+   -subj "/C=US/ST=California/L=San Francisco/O=Your Company/OU=Your Department/CN=*.$SYSTEM_DOMAIN" \
+   -addext "subjectAltName=DNS:$SYSTEM_DOMAIN,DNS:*.$SYSTEM_DOMAIN"
+```
+
+Validate that its contents are correct:
+
+```
+openssl x509 -noout -text -in 'cert.pem'
+```
+
+This should render something similar to this:
+
+```bash
+Certificate:
```

Comment:
Can be this identified as a potential data leak from infosec? (not sure just asking)

Reply:
It's a self generated certificate used only for the documentation (I run the command just to update the old one to be sure it was OK). I don't see any risk but I don't mind masking it if you want